src/helpers.cc

```diff
@@ -53,23 +53,6 @@ void log_info_target(const std::string& prefix, const Config& config, const Upta
   }
 }
 
-static void registerComposeAppManager(const std::shared_ptr<OSTree::Sysroot>& sysroot) {
```

Comment:
So I guess we never needed to register this, and the package manager already knew about it?

Reply:
n/m - i see  this isn't needed because we are basically doing our own factory logic now. makes sense.

Reply:
yes